### PR TITLE
quick fix for issue #194

### PR DESCRIPTION
**Pull request by luckman212**
### Status
**READY**

### Description
Small change related to #194 that opens the `URL` of an event in the default browser if it's not handled by `getMeetingLink()`

I think this improves the end user experience without causing any harm or unexpected functionality.

### Steps to Test or Reproduce

1. Create an event that has just a simple URL e.g. `https://www.google.com/search?q=MeetingBar`
image
2. Click on this event from MeetingBar

### CODE REVIEW

**Comment by luckman212** [comment]
Is there anything I can (or need) to do @leits to request a review of this?

File: MeetingBar/Helpers.swift [hunk 1]
```diff
@@ -189,6 +190,9 @@ func openEvent(_ event: EKEvent) {
             }
         }
         openMeetingURL(meeting.service, meeting.url, nil)
+    } else if eventUrl != "" {
+        // NSLog("opening \(eventUrl) in default browser")
+        event.url?.openInDefaultBrowser()
```
Comment (leits) [inline MeetingBar/Helpers.swift:195]:
```suggestion
    } else if let eventUrl = event.url {
        eventUrl.openInDefaultBrowser()
```

Looks like you can do this all in one string.
Please check this!

**Review by leits — COMMENTED** [review]
Hi @luckman212,
Thanks for the request!
Looks like makes sense, but please check my comment

File: MeetingBar/Helpers.swift (hunk 1 above)
Comment (luckman212) [reply MeetingBar/Helpers.swift]: Neat trick! @leits 
Looks like you're right of course. I committed that fix, tested & works.

**Review by leits — APPROVED** [review]

**Pull request merged**